packages/auth/src/grant/routes.test.ts

```diff
@@ -119,24 +119,35 @@ describe('Grant Routes', (): void => {
 
       describe('non-interactive grants', () => {
         describe.each`
-          interactionFlagsEnabled | description
-          ${true}                 | ${'enabled'}
-          ${false}                | ${'disabled'}
+          resourceInteractionFlagsEnabled | allGrantInteractionFlagEnabled | allGrant | description
+          ${true}                         | ${false}                       | ${false} | ${'no -all grant'}
+          ${false}                        | ${false}                       | ${false} | ${'no -all grant'}
+          ${true}                         | ${false}                       | ${true}  | ${'including an -all grant'}
+          ${false}                        | ${false}                       | ${true}  | ${'including an -all grant'}
+          ${true}                         | ${true}                        | ${true}  | ${'including an -all grant'}
+          ${false}                        | ${true}                        | ${true}  | ${'including an -all grant'}
```

Comment:
I _believe_ we are missing 2 cases here to capture all of the possibilities (8)?

Reply:
There are but having the flag enabled but no list-all grant isn't really worth testing, I think.